epispot/pre.py

```diff
@@ -2,7 +2,9 @@
 The 'pre-compiled' module contains already compiled models which can be put to use immediately.
 Each function returns an epispot Model object and its corresponding functions. Models parameters can still be changed
 even after compilation.
+
 STRUCTURE:
```

Comment:
Bold the `STRUCTURE` label, or make it a heading. **`STRUCTURE`** or
 ### Structure

Reply:
Done! :heavy_check_mark: